config/snapshotgc/snapshotgc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: snapshot-garbage-collector
+spec:
+  schedule: "* * * * *" # FIXME
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: test-gc
+              image: >-
+                quay.io/redhat-user-workloads/rhtap-integration-tenant/integration-service/integration-service@sha256:7e803efa5aa9561d5869096d3642106d0dae1567831e1ed6f2b0c7ac3e37fdcb
+              command:
+                - /snapshotgc
+                - --zap-log-level=debug
+                - --pr-snapshots-to-keep=1
+                - --non-pr-snapshots-to-keep=1
```

Comment:
@MartinBasti what do you think we should set for those?

Reply:
Of course. We decided on `M` and `N` :)

Reply:
totally not 1 :D

I'd suggest to start with something big and then lowering it

Reply:
also I think that it would be better to make it configurable, so for keep  limit we don't have to change integration service codebase everytime. Maybe use envvars and make them configurable via kustomize in infra-deploymets

Reply:
1024

Reply:
but quota is per both PR and push snapshots together 1024 

Let's start with 700 for push snapshots and 100 for PR snapshots, PR snapshots are basically throwaway

Reply:
There is still this part, how to change limits easier than updating integration-service and do full release round

Reply:
Do you have anything in mind?

Reply:
To unblock testing, I can waive it now, but it's something I'd like to resolve in next PRs

Reply:
I was suggesting env vars

Reply:
How would you change the value of the env var without having a release cycle?

Reply:
we can patch it using kustomize in infra deplyomets, without touching integration-service repo

Reply:
https://stackoverflow.com/questions/69828954/add-environment-variable-to-kubernetes-statefulset-with-kustomize something like this

Reply:
I see. Alright.